skrobot/planner/sqp_based.py

```diff
@@ -0,0 +1,164 @@
+import numpy as np
+from repoze.lru import lru_cache
```

Comment:
In python3, is `functools.lru_cache` faster than `repoze.lru.lru_cache`?
https://gist.github.com/iory/e82dab918c0a5c32cd3b1a4d1c06adbf

Reply:
Thank you for your time to giving me a benchmark. I fixed it in this commit :https://github.com/iory/scikit-robot/pull/181/commits/27bde96bea67cbe46312f3f4297e353c43fe035e